core/src/main/scala/kafka/server/DynamicConfig.scala

```diff
@@ -78,23 +78,42 @@ object DynamicConfig {
     def isQuotaConfig(name: String): Boolean = configNames.contains(name)
   }
 
+  object ClientConfigs {
+    val AcksOverrideProp = "acks"
+    val SessionTimeoutOverrideProp = "session.timeout.ms"
+    val HeartbeatIntervalOverrideProp = "heartbeat.interval.ms"
```

Comment:
Duplicate strings. These are specified somewhere else. Those symbols should be accessible from there.